Formula/gf.rb

```diff
@@ -0,0 +1,23 @@
+class Gf < Formula
+  desc "GoFrame is a modular, powerful, high-performance app dev framework of Golang"
+  homepage "https://goframe.org"
+  url "https://github.com/gogf/gf/archive/refs/tags/v2.2.2.tar.gz"
+  sha256 "6c289efa8ec11a035d350c61788ae041d74b0e61142bec87ea3461f01c16f88e"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    cd "cmd/gf" do
+      system "go", "build", *std_go_args(ldflags: "-s -w")
+    end
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/gf"
```

Reply:
this change cannot work because the go.mod under root path does not contain enough info for this tool. there is another go.mod under cmd/gf folder for build it.